### DISCUSSION TITLE
Download CI/ALPHA build.

### DISCUSSION THREAD

**Discussion by max5354** [z-huang/music#113] [General]
How to download your CI/Alpha Builds. I want to test that.

**Comment by z-huang** [comment]
To get a dev build, go to [GitHub Action](https://github.com/z-huang/music/actions) and download the APK artifact of any workflow.

**Reply by vedxp** [reply]
When should we expect build APKs in Repo Releases section?

**Reply by z-huang** [reply]
When the amount of update is big enough. It also depends on the speed of my work progress. I'm currently working on a new API library to communicate with YouTube. It includes some structure refactor. It's a big update and needs more time to complete.
